qiskit/transpiler/__init__.py

```diff
@@ -96,7 +96,8 @@
 
    .. jupyter-execute::
 
-      qc_basis = qc.decompose()
+      from qiskit.compiler import transpile
+      qc_basis = transpile(qc, basis_gates=backend.configuration().basis_gates)
```

Comment:
I think it would be better to just pass the `backend` instead of only the basis gates. That will also take into account e.g. the coupling map and insert swaps if necessary. 
```suggestion
      qc_basis = transpile(qc, backend)
```